src/sctools/platform.py

```diff
@@ -260,6 +260,44 @@ def calculate_gene_metrics(cls, args: Iterable[str] = None) -> int:
         gene_metric_gatherer.extract_metrics()
         return 0
 
+    @classmethod
+    def calculate_gene_metrics_fast(cls, args: Iterable[str] = None) -> int:
```

Comment:
Is there a non-fast method we can deprecate now? (or replace with this)

Reply:
deprecated.